packer/studio-selfhosted.pkr.hcl

```diff
@@ -120,6 +120,11 @@ build {
     destination = "/home/ubuntu/.studio_install/helm3.sh"
   }
 
+  provisioner "file" {
+    source      = "export-studio-logs.sh"
+    destination = "/usr/local/bin/export-studio-logs"
```

Comment:
I fixed the permission for GHA and there is 
```
==> amazon-ebs.source: Upload failed: scp: /usr/local/bin/export-studio-logs: Permission denied
```

https://github.com/iterative/studio-selfhosted/actions/runs/4606313140/jobs/8140343409?pr=48

Reply:
Oh, I forgot that writing to `/usr/local/bin` requires root permission.

From the Packer file provisioned docs:
> Warning: You can only upload files to locations that the provisioning user (generally not root) has permission to access. Creating files in /tmp and using a shell provisioner to move them into the final location is the only way to upload files to root owned locations.

Pushed a commit which hopefully fixes it